docker/kernel/httploader/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:alpine AS build
+
+RUN apk --no-cache add build-base git bzr mercurial gcc ca-certificates
+
+ADD ./httploader /src
+RUN cd /src && go get golang.org/x/sys/unix && CGO_ENABLED=0 GOOS=linux go build -a -o httploader -ldflags '-extldflags "-static"' .
```

Comment:
We need to use go modules

Reply:
Hey @kris-nova no we do not still have docs around Go since this is the first time Go language is in the play (and I'm really happy about this!).

BTW we have an issue for various technical docs and conventions we'd need. It's the issue https://github.com/falcosecurity/falco/issues/619